Files/style.css

```
@@ -0,0 +1,244 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 100%;
+}
+
+body {
+  font-family: sans-serif;
+  font-size: 0.9rem;
+  font-weight: 3;
+  position: relative;
+  background: #f5f5f5;
+  background-color: #eceef1;
+}
+
+nav {
+  margin-top: 10px;
+}
+
+.already {
+  float: right;
+  overflow: hidden;
+  margin-right: 20px;
+  font-size: 14px;
+  color: #757575;
+}
+
+.recovery-link {
+  float: right;
+  overflow: hidden;
+  margin-right: 20px;
+  text-decoration: none;
+  color: #0077c5;
+  font-size: 14px;
+}
+
+.sign-in-button {
+  margin-right: 20px;
+  float: right;
+  overflow: hidden;
+  color: #fff;
+  height: 35px;
+  width: 100px;
+  background: #0077c5;
+  border-radius: 4px;
+  border: 1px solid #4c4c4c;
+}
+
+.logo {
+  display: block;
+  width: 118px;
+  margin: 40px auto 18px;
+  padding-top: 20px;
+}
+
+.logos-under {
+  text-align: center;
+  margin: 15px auto;
+  justify-content: center;
+}
+
+.form-container {
+  background-color: #fff;
+  margin: auto;
+  border: 1px solid #c7c7c7;
+  width: 380px;
+  text-align: center;
+  color: #4c4c4c;
+  font-family: 'Nunito Sans', sans-serif;
+  padding-top: 30px;
+}
+
+.form-action {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.label-info {
+  display: inline-block;
+  width: 150px;
+  text-align: left;
+  margin-bottom: 8px;
+}
+
+#icons-container {
+  width: 100%;
+  height: 42px;
+  background-color: transparent;
+  text-align: center;
+}
+
+.image-icon {
+  width: 23px;
+  height: 23px;
+  margin-right: 4px;
+}
+
+#mint-icon {
+  width: 37px;
+  height: 20px;
+}
+
+#quick-books-icon {
+  width: 98px;
+  height: 22px;
+}
+
+#turbo-tax-icon {
+  width: 74px;
+  height: 22px;
+}
+
+input,
+textarea {
+  font-family: sans-serif;
+  font-size: 1em;
+  width: 300px;
+  box-sizing: border-box;
+  border: 1px solid #e0e0e0;
+}
+
+.input-info {
+  height: 30px;
+}
+
+input,
+input:focus {
+  border-color: #e0e0e0;
+}
+
+textarea,
+textarea:focus {
+  background-color: #e0e0e0;
+  vertical-align: top;
+  height: 5em;
+}
+
+label {
+  position: relative;
+}
+
+h1 {
+  font-size: 1.6em;
+  text-align: center;
+}
+
+p {
+  text-align: center;
+  font-size: 0.9em;
+}
+
+h5 {
+  font-size: 0.6em;
+  margin-top: 8px;
+  margin-bottom: 25px;
+  text-align: justify-all;
```

That is not a valid value for this property. I guess that you wanted to use `justify` instead.
```suggestion
  text-align: justify;
```